docs/pages/admin-guides/access-controls/device-trust/device-management.mdx

```diff
@@ -230,8 +222,8 @@ Device "c9cbb327-68a8-497e-b820-6a4b2bf58269" removed
 We recommend creating a dedicated `device-admin` role for device inventory
 management.
 
-<Admonition type="tip" title="v13.3.6+">
-  Teleport version 13.3.6 and above has the preset `device-admin` role, which
+<Admonition type="tip" title="Preset Device Administration role">
+  Teleport has the preset `device-admin` role, which
   is a substitute for the role described below.
```

Comment:
Let's remove the role below. We're asking the reader to create a role that already exists. This is not necessary.